<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportTabWidget</class>
 <widget class="QWidget" name="ReportTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Tab">
   <item>
    <widget class="QLabel" name="veritabaniLabel">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bağlantı bekleniyor...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="commentLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgb(240, 240, 240); 
        border-radius: 5px; 
        padding: 5px;
        color: #333;</string>
     </property>
     <property name="text">
      <string>Yorum alanı...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Dates">
     <item>
      <widget class="QLabel" name="labelBaslangic">
       <property name="text">
        <string>Başlangıç Tarihi:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="date_Baslangic">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBitis">
       <property name="text">
        <string>Bitiş Tarihi:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="date_Bitis">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>1</month>
         <day>31</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_Dates">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="btn_ApplyTemplate">
       <property name="text">
        <string>Taslağı Uygula</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Excel">
       <property name="text">
        <string>Excel'e Aktar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_PDF">
       <property name="text">
        <string>PDF'e Aktar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_Buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTaslak">
       <property name="text">
        <string>Rapor Taslağı:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="templateSecCBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="tarihSecCBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="geriTarihButton">
       <property name="text">
        <string>&lt; Önceki</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ileriTarihButton">
       <property name="text">
        <string>Sonraki &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tbl_Veri">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>